src/Periphery/LiFiDEXAggregator.sol

```diff
@@ -184,7 +189,9 @@ contract LiFiDEXAggregator is WithdrawablePeriphery {
                     if (step == 0) realAmountIn = usedAmount;
                 } else if (commandCode == 4) processOnePool(stream);
                 else if (commandCode == 5) processInsideBento(stream);
-                else if (commandCode == 6) applyPermit(tokenIn, stream);
+                else if (commandCode == 6)
```

Comment:
Could you please explain the rationale behind this code and why it had to be added (and maybe add it as a comment)?

Will Velodrome will only support tokens that support EIP2612?

Reply:
I didn't add it. It was just formatting. 

Regarding the permit functionality for velodrome the dex itself doesn't support EIP2612 tokens. Only LP tokens can have gasless approvals.

Reply:
ahhh...lol...OK, did not notice that, sorry.